charms/kfp-metadata-writer/tests/unit/test_operator.py

```diff
@@ -46,7 +46,7 @@ def test_kubernetes_component_created(harness, mocked_lightkube_client):
 
     # Assert that expected amount of apply calls were made
     # This simulates the Kubernetes resources being created
-    assert mocked_lightkube_client.apply.call_count == 3
```

Comment:
If removing the auth manifests means this charm is no longer creating k8s resources, then we should completely remove the k8s component. Note that it should be done in the roles removal PR as per my other comment.